class-workflow.php

```diff
@@ -372,6 +297,42 @@ public function update_all_module_options( $mod_name, $new_options ) {
 		$this->$mod_name->module           = $this->modules->$mod_name;
 		return update_option( $this->options_group . $mod_name . '_options', $this->modules->$mod_name->options );
 	}
+
+
+	/**
+	 * Collect all of the active post types
+	 *
+	 * @return array $post_types All of the post types that are 'on'
+	 */
+	public function get_supported_post_types(): array {
+		$post_types = [];
+		$post_types_options = $this->settings->module->options->post_types;
+
+		foreach ( $post_types_options as $post_type => $value ) {
+			if ( 'on' === $value ) {
+				$post_types[] = $post_type;
+			}
+		}
+
+		return $post_types;
+	}
+
+	/**
+	 * Whether or not the current page is our settings view. Determination is based on $pagenow, $_GET['page'], and if it's settings module or not.
+	 *
+	 * @return bool $is_settings_view Return true if it is
+	 */
+	public static function is_settings_view_loaded( string $slug ): bool {
```

Comment:
This was also moved from `Module` and simplified. We no longer check if the page matches any module slug, as this caused all VIP Workflow dependencies to be loaded on all VIP Workflow pages. Instead, we pass the current module `$slug` and directly check if the current page is loaded.